tests/conftest.py

```diff
@@ -202,3 +202,61 @@ def persisted_task(request):
 def test_assets_directory() -> pathlib.Path:
     """Gets path to test_assets directory."""
     return pathlib.Path(__file__).parent / "test_assets"
+
+
+@pytest.fixture(scope="module")
+def ai_voltage_chan_with_excit(any_x_series_device):
+    """Creates AI Channel object to measure Voltage.
+
+    Used in testing channel property of uint32 and boolean data types.
+    """
+    task = nidaqmx.Task()
```

Comment:
Wait, we're leaking these tasks. It looks like using a [yield fixture](https://docs.pytest.org/en/6.2.x/fixture.html#teardown-cleanup-aka-fixture-finalization) is the right way to clean this up. You can put this in a `with` block and yield the `ai_channel`

Reply:
Updated the fixture accordingly. Thanks for the suggestion Zach :)